src/Compilers/VisualBasic/Portable/Declarations/DeclarationTreeBuilder.vb

```diff
@@ -220,6 +230,66 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End If
         End Function
 
+        Private Shared Function GetAliasedQuickAttributes(compilationUnit As CompilationUnitSyntax) As QuickAttributes
```

Comment:
>GetAliasedQuickAttributes [](http://example.com/codeflow?start=32&length=25)

nit: "Aliased" versus "Imported"?

Reply:
'Aliased' is appropriate in this case.  in VB it's an import-alias corresponding to C#'s using-alias.